docs/whatsnew/3.0.rst

```diff
@@ -38,3 +38,56 @@ Full change log
 
 To see a detailed list of all changes in version v3.0, including changes in
 API, please see the :ref:`changelog`.
+
+
+Renamed/removed functionality
+=============================
+
+pytest plugins
+**************
+
+The following pytest plugins were previously provided as part of the Astropy
```

Comment:
In the diff above, there are double-backticks around pytest to render it as `pytest`; Should we be consistent here?